Listenbrainz/Model/Feed/FeedAdditionalInfo.swift

```diff
@@ -5,50 +5,16 @@
 //  Created by Gaurav Bhardwaj on 31/08/23.
 //
 
-//import Foundation
-//
-//
-//
-//struct FeedAdditionalInfo: Codable {
```

Comment:
@gauravbhardwaj7 is this correct? This breaks the feed I think

Reply:
Nah @akshaaatt , I have been using these parameters, recording_msid and mbid in pinning, recommending and originUrl for opening in spotify

Reply:
Check that you have commented out this entire file @gauravbhardwaj7

Reply:
yeah, I have removed the whole file and then added the required fields in the additional info model.
`import Foundation

struct FeedAdditionalInfo: Codable {
    let originURL: String?
    let recordingMbid: String?
    let recordingMsid: String

    enum CodingKeys: String, CodingKey {
        case originURL = "origin_url"
        case recordingMbid = "recording_mbid"
        case recordingMsid = "recording_msid"
    }
}` - please check whether you are able to access these loc, @akshaaatt

Reply:
@gauravbhardwaj7 can you remove the commented code?

Reply:
Ahh, it's removed @akshaaatt , only above loc are there in this model file currently, can you please check again :(

Reply:
Argh! My bad! :)